### ISSUE TITLE
[Truffle] Issue with getting backtick output

### ISSUE THREAD

**Issue by bjfish** [jruby/jruby#2889]
Expected
```
ruby -e 'puts `/Users/brandonfish/Documents/jruby/bin/jruby -J-ea -J-Xmx2G -e "puts true"`'
true
```

Actual
```
 ruby -e 'puts `/Users/brandonfish/Documents/jruby/bin/jruby -X+T -J-ea -J-Xmx2G -e "puts true"`'

```

**Comment by bjfish** [comment]
This is need for the argf tests.

**Comment by chrisseaton** [comment]
The problem is that our `File` objects for stdout and stderr are not flushed on exit, as they're just normal objects - the runtime doesn't know anything about them. Minimal example of the problem:

```
$ jt e print 14 | wc
       0       0       0
```

**Comment by chrisseaton** [comment]
Fixed in 9e518a1ba1a3614e58f1b5c736994c01780161dc
